lib/fog/kubevirt.rb

```diff
@@ -0,0 +1 @@
+require 'fog/kubevirt/compute'
```

Comment:
this should be modified to `fog/compute/kubevirt'` to match https://github.com/fog/fog-kubevirt/blob/master/lib/fog/compute/kubevirt.rb